exporters/zipkin/zipkin.go

```diff
@@ -77,30 +87,36 @@ func WithClient(client *http.Client) Option {
 }
 
 // New creates a new Zipkin exporter.
-func New(collectorURL string, opts ...Option) (*Exporter, error) {
-	if collectorURL == "" {
-		return nil, errors.New("collector URL cannot be empty")
+func New(opts ...Option) (*Exporter, error) {
```

Comment:
While I think I would prefer this interface, this module is stable at v1.3.0 and we cannot change its interface in this way.  What about just removing the error if `collectorURL` is empty?